<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ParkingApp</class>
 <widget class="QMainWindow" name="ParkingApp">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1194</width>
    <height>610</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>0</y>
      <width>20</width>
      <height>521</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLabel" name="camera">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>341</width>
      <height>211</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="Line" name="line_3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>731</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="camera2">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>60</y>
      <width>341</width>
      <height>211</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="camera4">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>310</y>
      <width>341</width>
      <height>211</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="camera3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>310</y>
      <width>341</width>
      <height>211</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QToolBox" name="toolBox">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>0</y>
      <width>431</width>
      <height>551</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="page">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>431</width>
       <height>464</height>
      </rect>
     </property>
     <attribute name="label">
      <string>SettingPage</string>
     </attribute>
     <widget class="QTabWidget" name="tabWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>401</width>
        <height>421</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Calibration</string>
       </attribute>
       <widget class="QTextEdit" name="recording_folder">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>361</width>
          <height>31</height>
         </rect>
        </property>
        <property name="html">
         <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Sans Serif'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;test&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
       <widget class="QCheckBox" name="record_type_btn">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>50</y>
          <width>85</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>record</string>
        </property>
       </widget>
       <widget class="QPushButton" name="recordBtn">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>80</y>
          <width>101</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Not Recording</string>
        </property>
       </widget>
       <widget class="QCheckBox" name="calibrateBtn">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>150</y>
          <width>101</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Calibrated</string>
        </property>
       </widget>
       <widget class="QPushButton" name="openCalibrationBtn">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>180</y>
          <width>41</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Open</string>
        </property>
       </widget>
       <widget class="QPushButton" name="saveCalibrateBtn">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>180</y>
          <width>41</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Save </string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Data</string>
       </attribute>
       <widget class="QTextEdit" name="TestingDataInput">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>291</width>
          <height>41</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>Networking</string>
       </attribute>
       <widget class="QTextEdit" name="WebLocationInput">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>291</width>
          <height>41</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="monitorBtn">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>70</y>
          <width>101</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Monitor start</string>
        </property>
       </widget>
       <widget class="QPushButton" name="openBtn">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>120</y>
          <width>101</width>
          <height>31</height>
         </rect>
        </property>
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;open the url of the IP camera&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="text">
         <string>Open Camera</string>
        </property>
       </widget>
       <widget class="QPushButton" name="saveVideoBtn">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>170</y>
          <width>111</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>start Video</string>
        </property>
       </widget>
       <widget class="QLabel" name="statusField">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>210</y>
          <width>61</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>OFF</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>210</y>
          <width>61</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Status:</string>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="TestingData">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>431</width>
       <height>464</height>
      </rect>
     </property>
     <attribute name="label">
      <string>TestingPage</string>
     </attribute>
     <widget class="QListView" name="videoList">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>100</y>
        <width>411</width>
        <height>341</height>
       </rect>
      </property>
     </widget>
     <widget class="QTextEdit" name="ipField">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>381</width>
        <height>41</height>
       </rect>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Sans Serif'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; color:#008000;&quot;&gt;http://192.168.1.126:81/videostream.asf?user=admin&amp;amp;pwd=&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="loadBtn">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Load Local File</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>50</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Load Image Files</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Recording">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>50</y>
        <width>80</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Recording</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="DetectorSetting">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>431</width>
       <height>464</height>
      </rect>
     </property>
     <attribute name="label">
      <string>DetectorPage</string>
     </attribute>
     <widget class="QTabWidget" name="HOG">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>461</width>
        <height>461</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="PixelTab">
       <attribute name="title">
        <string>Pixel</string>
       </attribute>
       <widget class="QLineEdit" name="pixel_threshold_input">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>30</y>
          <width>101</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>0.5</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>81</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Threshold</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_5">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>80</y>
          <width>101</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Updating Rate</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="pixel_updaterate_input">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>80</y>
          <width>101</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>0.5</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="EdgeTab">
       <attribute name="title">
        <string>Edge</string>
       </attribute>
       <widget class="QComboBox" name="edge_algorithms_list">
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>20</y>
          <width>141</width>
          <height>23</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="edge_threshold_input">
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>60</y>
          <width>101</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>0.5</string>
        </property>
       </widget>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>20</y>
          <width>81</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Algorithms</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>60</y>
          <width>81</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Threshold</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="HOGTab">
       <attribute name="title">
        <string>HOG</string>
       </attribute>
       <widget class="QPushButton" name="training_btn">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>120</y>
          <width>111</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Train Vehicle</string>
        </property>
       </widget>
       <widget class="QPushButton" name="vehicleDetectBtn">
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>120</y>
          <width>111</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Detect Vehicle</string>
        </property>
       </widget>
       <widget class="QPushButton" name="personDetectBtn">
        <property name="geometry">
         <rect>
          <x>260</x>
          <y>120</y>
          <width>121</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Detect Person</string>
        </property>
       </widget>
       <widget class="QGraphicsView" name="graphicsView">
        <property name="geometry">
         <rect>
          <x>220</x>
          <y>170</y>
          <width>181</width>
          <height>141</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_6">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>81</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Window Size</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="hog_window_size_input">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>10</y>
          <width>101</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>64</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_7">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>40</y>
          <width>81</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Block Size</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="hog_block_size_input">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>40</y>
          <width>101</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>16</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_8">
        <property name="geometry">
         <rect>
          <x>220</x>
          <y>10</y>
          <width>81</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Block Stride</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="block_stride_input">
        <property name="geometry">
         <rect>
          <x>300</x>
          <y>10</y>
          <width>101</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>16</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_9">
        <property name="geometry">
         <rect>
          <x>220</x>
          <y>40</y>
          <width>81</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Cell Size</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="cell_size_input">
        <property name="geometry">
         <rect>
          <x>300</x>
          <y>40</y>
          <width>101</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>16</string>
        </property>
       </widget>
       <widget class="Line" name="line_2">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>100</y>
          <width>411</width>
          <height>16</height>
         </rect>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="DPMTab">
       <attribute name="title">
        <string>DPM</string>
       </attribute>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="forwardBtn">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>0</y>
      <width>80</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>forward</string>
    </property>
   </widget>
   <widget class="QPushButton" name="videoFrameBtn">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>0</y>
      <width>41</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Jump</string>
    </property>
   </widget>
   <widget class="QLabel" name="frameCurrentLabel">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>0</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QTextEdit" name="frameSelectInput">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>0</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Sans Serif'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="videoControlBtn">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>0</y>
      <width>80</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Play</string>
    </property>
   </widget>
   <widget class="QLabel" name="frameLabel11">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>0</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>/</string>
    </property>
   </widget>
   <widget class="QPushButton" name="backwardBtn">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>0</y>
      <width>80</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>backward</string>
    </property>
   </widget>
   <widget class="QLabel" name="frameTotalLabel">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>0</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>0</y>
      <width>118</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>40</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Original View</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>40</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Edge based View</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>280</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Pixel Based View</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>280</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>HOG Based View</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1194</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuPrototype_environment">
    <property name="title">
     <string>Prototype environment</string>
    </property>
   </widget>
   <addaction name="menuPrototype_environment"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
